FirebaseAppDistribution/Sources/Public/FIRAppDistribution.h

```diff
@@ -0,0 +1,110 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "FIRAppDistributionRelease.h"
+
+#import <AppAuth/AppAuth.h>
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ *  @related AppDistributionError
+ *
+ *  The completion handler invoked when the new build request returns.
+ *  If the call fails we return the appropriate `error code`, described by
+ *  `AppDistributionError`.
+ *
+ *  @param release  The new release that is available to be installed.
+ *  @param error     The error describing why the new build request failed.
+ */
+typedef void (^FIRAppDistributionUpdateCheckCompletion)(
+    FIRAppDistributionRelease *_Nullable release, NSError *_Nullable error)
+    NS_SWIFT_NAME(AppDistributionUpdateCheckCompletion);
+
+/**
+ *  @related AppDistributionError
+ *
+ *  The completion handler invoked  when App Distribution sign in is complete
+ *  If the call fails we return the appropriate `error code`, described by
+ *  `AppDistributionError`.
+ *
+ *  @param error     The error describing why sign in failed
+ */
+typedef void (^FIRAppDistributionSignInTesterCompletion)(NSError *_Nullable error)
+    NS_SWIFT_NAME(AppDistributionSignInTesterCompletion);
```

Comment:
This should be removed: in the API proposal we decided to exclude this and just use the raw error block in the method definition.

Reply:
done